pandas/core/nanops.py

```diff
@@ -1377,7 +1377,8 @@ def func(a, b):
 
 
 @disallow("M8", "m8")
-def nancov(a: np.ndarray, b: np.ndarray, min_periods: Optional[int] = None):
+def nancov(a: np.ndarray, b: np.ndarray, min_periods: Optional[int] = None,
+           ddof: Optional[int] = 1):
```

Comment:
Can you run black to get this to format correctly (also will need to do that for series.py)?

Reply:
Done. Thank you for your help.